### DISCUSSION TITLE
Asset Links / universal-links

### DISCUSSION THREAD

**Discussion by renatofreireda** [apache/cordova#378] [Q&A]
Hello everybody!
I would like someone's help to get universal-links asset links to work on cordova. I noticed that some plugins are already obsolete or discontinued. Can anyone help?

**Comment by breautek** [comment]
Moving this to cordova discussions as it doesn't relate to any bug or isuse within Cordova.

**Comment by dpogue** [comment]
https://github.com/AyogoHealth/cordova-plugin-app-scope is a plugin I wrote a few years ago to handle universal links and asset links. It's fairly simple: no JS API, just navigates the webview to the URL that comes in; but it has worked well for my use cases over the years.
